### DISCUSSION TITLE
Celebs roulette

### DISCUSSION THREAD

**Discussion by cwolin57** [aclap-dev/video-downloadhelper#338] [Help and Q&A]
### Your question

I cannot download vidoes from Celebsroulette.  When I try I get a message "no download item found". Can anyone help me with that?

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only 16 first characters): 8cd71b26620c4271
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "mac",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/Users/cwolin",
  "converterBinary": "/Applications/net.downloadhelper.coapp.app/Contents/MacOS/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by Mattmjs** [comment]
I get detection from that site, maybe you need to update to the latest companion app. Also did you play the media first.
Sometimes that is required before there is detection.

**Comment by cwolin57** [comment]
It has been a while but I have tried this again with no success.  I believe I have the latest companion app installed. I have played the media first.  I have attached a screenshot showing that the app has found the media.  However, when I try to download it, I get an error almost immediately that no media is found.  I would appreciate your help. I have also attached screenshot showing, what I hope is relevant data about what I have installed.

Thanks

> On Mar 28, 2024, at 3:08 AM, Mattmjs ***@***.***> wrote:
> 
> 
> I get detection from that site, maybe you need to update to the latest companion app. Also did you play the media first.
> Sometimes that is required before there is detection.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/aclap-dev/video-downloadhelper/discussions/338#discussioncomment-8936965>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BHMWVDOMJ2VIT6T4FOIKUNDY2O6VFAVCNFSM6AAAAABFMDQNCOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSMZWHE3DK>.
> You are receiving this because you authored the thread.
> 
￼￼￼

**Comment by cwolin57** [comment]
I just sent an email asking for help.  I was trying the download in Mozilla Firefox.  I tried it in Chrome and it worked so I don’t need further assistance.

Thanks
